server/real_estate/src/test/java/com/realestate/real_estate/repos/PropertyRepositoryTests.java

```diff
@@ -26,6 +26,7 @@ public class PropertyRepositoryTests {
     private PropertyRepository propertyRepository;
 
     // Arrange
+    String id = new String();
```

Comment:
Why do you need this?

Reply:
Honestly, I really don't know. It was a part of the unit test, I just changed it to String datatype. Should I remove it?

Reply:
Nah, if everything is working let's not touch it